api4/user.go

```diff
@@ -42,6 +43,8 @@ func InitUser() {
 	BaseRoutes.User.Handle("/sessions", ApiSessionRequired(getSessions)).Methods("GET")
 	BaseRoutes.User.Handle("/sessions/revoke", ApiSessionRequired(revokeSession)).Methods("POST")
 	BaseRoutes.User.Handle("/audits", ApiSessionRequired(getAudits)).Methods("GET")
+
+	BaseRoutes.Users.Handle("/autocomplete/", ApiSessionRequired(autocompleteUsers)).Methods("GET")
```

Comment:
I can make it work without a "/" by changing: 
BaseRoutes.User = BaseRoutes.Users.PathPrefix("/{user_id:[A-Za-z0-9]+}").Subrouter() to 
BaseRoutes.User = BaseRoutes.ApiRoot.PathPrefix("users/{user_id:[A-Za-z0-9]+}").Subrouter(). 

Let me know if you want me to do it like this